File: bots/betfair/betfairLogout.m
function betfairLogout(sessionKey,appKey)

% logs out of betfair API

%% Send data to betfair API via http:

% define url to send data to:
url = ['https://identitysso.betfair.com/api/logout'];

% define http-request headers:
headersIn(1).name = 'X-Authentication';
headersIn(1).value = sessionKey;
headersIn(2).name = 'X-Application';
headersIn(2).value = appKey;
headersIn(3).name = 'Accept';
headersIn(3).value = 'application/json';

% define the body of the http-request:
body='';

% send data to url:
[output,extras] = urlread2(url,'POST',body ,headersIn);

File: bots/betfair/betfairLogout.py
import requests

def betfairLogout(sessionKey,appKey):

    # logs out of betfair API

    ## Send data to betfair API via http:

    # define url to send data to:
    url = 'https://identitysso.betfair.com/api/logout'
    headers = {'X-Authentication': sessionKey, 'X-Application': appKey, 'Accept': 'application/json'}
    content = '{"jsonrpc": "2.0", "method": "AccountAPING/v1.0/getDeveloperAppKeys"}'
    response = requests.post(url, params=content, headers=headers)
    return response.json()
